wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -72,8 +72,10 @@ const Lobby = ({location}) => {
             console.log(top10)
             setGoSwipe(true);
         });
+        return () => {socket.off('sessionMembers')};
```

Comment:
thanks for adding this! this is definitely contributing to good practice as we move forward with using sockets! 💯